### DISCUSSION TITLE
Should vcpkg add supported version information for dependencies like CUDAToolkit?

### DISCUSSION THREAD

**Discussion by Bzz2745** [microsoft/vcpkg#38185] [Q&A]
When I reset the system and compile some third-party libraries that rely on CUDAToolkit, such as vtk, I found that even though I have set the environment variables of CUDA_PATH, CUDAToolkit_ROOT and path correctly, I still get the error: Failed to find nvcc. Compiler requires the CUDA toolkit.  Please set the CUDAToolkit_ROOT。 Shouldn't vcpkg start outputting the corresponding supported version when installing these third-party libraries that require CUDA?

**Comment by Neumann-A** [comment]
vcpkg should provide a cuda port which actually provides CUDA so that the user cannot shot him/herself into the foot. 
There is #35336 to download CUDA instead.
